ResultadosDigitais/matrix
# Adds i18n to login application and refactors google signin button

### Description

This PR extracts and translate the strings from the login application. I had to refactor the google signin button because it used static images which were impossible to translate, so I just did it with css mimicking the specs from the Google documentation.

resolves #341 

![image](https://user-images.githubusercontent.com/12252332/88247800-6bcc1c00-cc75-11ea-83cf-44c8b512487c.png)

### How to test?
If your OS language is `pt-br` you should see the login page in portuguese by default, otherwise you can login, change the language to portuguese and logout.

### Expected behavior
The login page should be in portuguese or english, according to the language choice/browser default.

---
This PR is pointing to the add-i18n branch because it came it, so as soon as `add-i18n` gets merged, this PR should point to develop, or just merge it to add-i18n before it gets merged to develop. There will be other PRs with the same needs.
